nrf_security/cmake/config_to_tf-m.cmake

```diff
@@ -34,3 +34,10 @@ if(CONFIG_TFM_BL2)
       -DMCUBOOT_MBEDCRYPTO_CONFIG_FILEPATH:STRING=${CONFIG_MBEDTLS_CFG_FILE}
   )
 endif()
+
+if(CONFIG_PSA_ITS_ENCRYPTED)
```

Comment:
Maybe there should be a separate TFM_ option for doing this, for easier discoverability. That option could also select the PSA_ITS_ENCRYPTED option.

Reply:
Let me see if I get it straight. You propose to have TFM_ITS_ENCRYPTED which selects PSA_ITS_ENCRYPTED?

Reply:
Yes, so that users are able to find this config among the TFM configs.

Reply:
Isn't it better to have the other way around? Meaning to have the PSA_ITS_ENCRYPTED select the TFM_ITS_ENCRYPTED? Otherwise  platforms will have to actively enable a different configurations based on what they support if I am not mistaken.

Reply:
I think that TFM_ITS_ENCRYPTED is the standardized config and PSA_ITS_ENCRYPTED is our convenience for handling non-TF-M

I would suggest TFM_ITS_ENCRYPTED selecting PSA_ITS_ENCRYPTED

Reply:
Should be done now ;)